### PR TITLE
link-grammar: init at 5.5.0

### PR DESCRIPTION
**Pull request by siers**
###### Motivation for this change

Hasn't been packaged.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

@jtojnar gave me this nix file and said he'd be okay with maintaining it, but didn't want to submit a PR himself, so here it is.

### CODE REVIEW

File: pkgs/tools/text/link-grammar/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, pkgconfig, python3, sqlite, libedit, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  version = "5.5.0";
```
Comment (jtojnar) [inline pkgs/tools/text/link-grammar/default.nix]: 5.5.1 is available

**Comment by jtojnar** [comment]
@GrahamcOfBorg build link-grammar

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -1421,6 +1421,8 @@ with pkgs;
 
   libxnd = callPackage ../development/libraries/libxnd { };
 
+  link-grammar = callPackages ../tools/text/link-grammar { };
```
Comment (matthewbauer) [inline pkgs/top-level/all-packages.nix]: Should be singular, callPackage

Comment (siers) [reply pkgs/top-level/all-packages.nix]: true, fixed

**Comment by siers** [comment]
@GrahamcOfBorg build link-grammar

**Pull request merged**